api/app/responses/api.py

```diff
@@ -458,3 +463,80 @@ def get(self, event_id):
         num_reviewers = review_config.num_reviews_required + review_config.num_optional_reviews if review_config is not None else 1
 
         return ResponseDetailAPI._serialize_response(response, language, review_form_id, num_reviewers)
+
+class ResponseExportAPI(restful.Resource):
+
+    def get(self):
+
+        def _get_answer(id, answers):
```

Comment:
Better to call this parameter `question_id` to make it clear which ID it's referring to (Answer's also have IDs)